### ISSUE TITLE
TypeError: undefined is not iterable, when TransitionLink in main layout

### ISSUE THREAD

**Issue by dodikuswono** [AndreasFaust/gatsby-plugin-transitions#5]
Hi,

Error happen when adding TransitionLink in main layout (src/layouts/index.js), but not gatsby Link :
> TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))

src/layouts/index.js :
```
import React from "react";
import { TransitionProvider, TransitionLink } from "gatsby-plugin-transitions";

const Layout = ({ location, children }) => {
    return (
        <>
            <TransitionProvider location={location} >{children}</TransitionProvider>
            <TransitionLink to='/news3'>News</TransitionLink>
        </>
    );
};

export default Layout;
```

Thanks.

**Comment by AndreasFaust** [comment]
Hello! Unfortunately you cannot use `TransitionLink` outside of `TransitionProvider` yet.

**Comment by AndreasFaust** [comment]
This problem got fixed in version `2.0.0`. **See the docs!**
